go/pkg/pwapi/api_admin-update-teams-metadata.go

```diff
@@ -0,0 +1,28 @@
+package pwapi
+
+import (
+	"context"
+
+	"pathwar.land/pathwar/v2/go/pkg/pwdb"
+
+	"pathwar.land/pathwar/v2/go/pkg/errcode"
+)
+
+func (svc *service) AdminUpdateTeamsMetadata(ctx context.Context, in *AdminUpdateTeamsMetadata_Input) (*AdminUpdateTeamsMetadata_Output, error) {
+	if !isAdminContext(ctx) {
+		return nil, errcode.ErrRestrictedArea
+	}
+
+	if in == nil || in.Teams == nil {
+		return nil, errcode.ErrMissingInput
+	}
+
+	for _, team := range in.Teams {
+		err := svc.db.Model(&pwdb.Team{}).Update(team).Error
```

Comment:
this is unsafe, can you specify the whitelisted fields that can be updated instead of taking the whole object?